### ISSUE TITLE
[feature] enable listing of commands by tag

### ISSUE THREAD

**Issue by masukomi** [corporate-runaways/TooLoo#10]
Builds on, and is blocked by Issue #9 (adding initial support for tags)

Not terribly useful on the command line, this could be a nice-to-have for exporting to create web pages. This would enable web users to browse by tag. 

Still, we'll start with the command line because it's the easiest way to implement the core functionality we'll need for the export. 

I'm thinking `clu list <tag>`  as a way of invoking this, but am open to other suggestions.
